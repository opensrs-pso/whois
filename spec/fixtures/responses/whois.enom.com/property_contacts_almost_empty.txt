=-=-=-=
Visit AboutUs.org for more information about johnmcenroetennisacademy.com
<a href="http://www.aboutus.org/johnmcenroetennisacademy.com">AboutUs: johnmcenroetennisacademy.com</a>

	
Domain name: johnmcenroetennisacademy.com

Registrant Contact:
   
   Tony Kerr ()
   
   Fax: 
   ,  
   US

Administrative Contact:
   
   Tony Kerr (tkerr@sportimeny.com)
   +1.
   Fax: 
   ,  
   US

Technical Contact:
   
   Tony Kerr (tkerr@sportimeny.com)
   +1.
   Fax: 
   ,  
   US

Status: Locked

Name Servers:
   ns1.saturn5.net
   ns2.saturn5.net
   
Creation date: 29 Apr 2010 17:18:00
Expiration date: 29 Apr 2020 12:18:00


   
Get Noticed on the Internet!  Increase visibility for this domain name by listing it at www.whoisbusinesslistings.com
=-=-=-=
The data in this whois database is provided to you for information
purposes only, that is, to assist you in obtaining information about or
related to a domain name registration record. We make this information
available "as is," and do not guarantee its accuracy. By submitting a
whois query, you agree that you will use this data only for lawful
purposes and that, under no circumstances will you use this data to: (1)
enable high volume, automated, electronic processes that stress or load
this whois database system providing you this information; or (2) allow,
enable, or otherwise support the transmission of mass unsolicited,
commercial advertising or solicitations via direct mail, electronic
mail, or by telephone. The compilation, repackaging, dissemination or
other use of this data is expressly prohibited without prior written
consent from us.  

We reserve the right to modify these terms at any time. By submitting 
this query, you agree to abide by these terms.
Version 6.3 4/3/2002
