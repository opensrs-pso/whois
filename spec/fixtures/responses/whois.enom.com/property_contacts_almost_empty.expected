#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].name         == "Tony Kerr"
  should: %s[0].organization == ""
  should: %s[0].address      == ""
  should: %s[0].city         == ""
  should: %s[0].zip          == ""
  should: %s[0].state        == ""
  should: %s[0].country_code == "US"
  should: %s[0].phone        == ""
  should: %s[0].fax          == ""
  should: %s[0].email        == ""

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].name         == "Tony Kerr"
  should: %s[0].organization == ""
  should: %s[0].address      == ""
  should: %s[0].city         == ""
  should: %s[0].zip          == ""
  should: %s[0].state        == ""
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1."
  should: %s[0].fax          == ""
  should: %s[0].email        == "tkerr@sportimeny.com"
